README.md

```diff
@@ -621,6 +621,79 @@ is received, allowing them to accept or reject it. A custom prompt that matches
   - When the user performs an action (i.e. tapping a button) that indicates they accept, the view controller should call `processAcceptance(payloadExtras: [:])` from the `AcceptancePromptViewController` superclass.
   - When the user performs an action that indicates they reject, the view controller should call `processRejection()` in the superclass.
 - To use the custom prompt, the `buildAcceptancePromptViewController()` method should be implemented in the UI delegate (described above), and it should return the custom `AcceptancePromptViewController` class.
+
+To use a custom Acceptance Prompt you have to assign a trusonaficationUIDelegate object to your Trusona Instance, like this:
+
+```swift
+let uiDelegate = MyClassImplementingUIDelegate()
+let trusona = Trusona(region: .asiaPacific)
+trusona.trusonaficationUIDelegate = uiDelegate
+```
+
+Now, you have to define your class that implements the *TrusonaficationUIDelegate* protocol
+
+```swift
+class MyClassImplementingUIDelegate: TrusonaficationUIDelegate {
+  func buildAcceptancePromptViewController() -> AcceptancePromptViewController {
+    
+    // The storyboard where we created our custom Accept and Reject ViewController
+    let myStoryboard = UIStoryboard(name: "MyStoryboard", bundle: nil)
+    
+    // Instantiate our ViewController with the custom class we created
+    let myAcceptRejectController = myStoryboard.instantiateViewController(withIdentifier: "MyAcceptRejectViewController") as! MyAcceptRejectViewController
+    
+    return myAcceptRejectController
+  }
+}
+```
+
+As you can see in the code before, we are instantiating a View Controller that has our custom UI and then returning it with the buildAcceptancePromptViewController method
```

Comment:
`buildAcceptancePromptViewController` might be easier to read with the inline quote tags (instead of just "buildAcceptancePromptViewController")